Jenkinsfile

```diff
@@ -1,20 +1,25 @@
-@Library('socrata-pipeline-library@5.0.0') _
+@Library('socrata-pipeline-library@generalize-sbt-to-work-for-multiple-projects') _
```

Comment:
Needs to be updated to the pipelines version

Reply:
Yes, I just haven't updated since the branch was merged.